app/models/query.rb

```diff
@@ -62,6 +62,7 @@ def self.new_default(attributes = nil)
       query.add_default_filter
       query.set_default_sort
       query.show_hierarchies = true
+      query.include_subprojects = Setting.display_subprojects_work_packages?
```

Comment:
Ideally this would now all be moved to the `SetAttributeService` since it should handle default value setting completely. But there seem to be too many places where `.new_default` is used to do that now.